### ISSUE TITLE
Nice page

### ISSUE THREAD

**Issue by UlisesAC4** [azaroma/magmahackers-task-1#3]
I love materializecss! hahaha

I do not really see problems in your code, it is not worth to use a separate file for those styles you are using.

Adding comments should be nice to know which code belongs to which parts.

It seems that we need more practice anyways greetings :)

**Comment by azaroma** [comment]
Yes, you're right, style in html file is really bad practice. I intended to write CSS for that but I forgot. Thank you for your review. What do you prefer Bootstrap or Materialize?
